CSE115A/rideshare-project
# fixed autocomplete bug

Problem & Solution 
=======
Google Maps API has to be hosted server-side, so we created an autocomplete function on Firebase and now we have fixed the front end to call our API endpoint. 

[Link to Ticket](https://trello.com/c/K4JJZvta)

Change summary:
---------------
* fixed autocomplete bug e81243d

Steps to Verify:
----------------
1. Checkout this branch and add your env variable to hit the prod endpoint of autocomplete 
1. Verify that when typing, it populates the dropdown with suggestions.
